### PR TITLE
Component/TagGroup

### PR DESCRIPTION
**Pull request by christophschmitz**
Hello, I worked on my TagGroup component.

- Add TagGroup component
- Add styling to TagGroup component

[Deployment link](https://mastery-component-taggr-jnm4av.herokuapp.com/storybook/?path=/story/component-tag--taggroup)

### CODE REVIEW

File: src/app/components/TagGroup/TagGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Tag from '../Tag/Tag';
+import styles from './TagGroup.module.css';
+
+type TagProps = {
```
Comment (phlgr) [inline src/app/components/TagGroup/TagGroup.tsx:5]:
```suggestion
type Tag = {
```

File: src/app/components/TagGroup/TagGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export type TagGroupProps = {
+  tagList: TagProps[];
+  className?: string;
+};
+
+const TagGroup = ({ tagList, className }: TagGroupProps): JSX.Element => {
+  return (
+    <section className={`${styles.container} ${className}`}>
+      {tagList.map((tag, index) => (
+        <Tag key={index} active={tag.active} onClick={tag.onClick}>
```
Comment (phlgr) [inline src/app/components/TagGroup/TagGroup.tsx:21]: What does the onClick function do? Does it need to know which button got clicked?

Comment (phlgr) [inline src/app/components/TagGroup/TagGroup.tsx:21]: Please don't use the index as key, rather use the children if it is unique

https://reactwithhooks.netlify.app/docs/lists-and-keys.html#keys

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/TagGroup/TagGroup.tsx (hunk 2 above)
Comment (christophschmitz) [reply src/app/components/TagGroup/TagGroup.tsx:21]: Yes, I want to use them as a filter.

**Pull request merged**